_posts/2015-04-17-0.36-release.md

```diff
@@ -0,0 +1,17 @@
+---
+title:  "Release 0.36"
+author: Julien Pagès
+---
+
+This release includes a bugfix for the download in backgound feature introduced
+in 0.35 (see [bug 1153801]).
+
+The fennec profile is also taken in account now, meaning:
```

Comment:
A few changes I'd make:

The fennec profile is also taken in account now, meaning: -> Also, on Firefox for Android we now use a custom profile, meaning:

(Firefox for Android means something to more people than Fennec)